python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -30,7 +30,7 @@
 
 _imported_GangaList = None
 
-do_not_copy = ['_proxyObject', '_proxyClass', '_data', '_index_cache', '_parent', '_registry', '_impl']
+do_not_copy = ['_proxyObject', '_data', '_index_cache', '_parent', '_registry']
```

Comment:
Yes, it's safe to copy the `_proxyClass` itself since a class copy (as opposed to an object copy) does not recurse any deeper. `_impl` was removed since it's never present in a `Node` or `GangaObject`.